com.github.antimicro.json

```
@@ -0,0 +1,43 @@
+{
+    "app-id": "com.github.antimicro",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "antimicro",
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-config/antimicro",
+        "--env=XDG_CONFIG_HOME=~/.config"
+    ],
+    "modules": [
+        {
+            "name" : "sdl2",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.libsdl.org/release/SDL2-2.0.9.tar.gz",
+                    "sha256" : "255186dc676ecd0c1dbf10ec8a2cc5d6869b5079d8a38194c2aecdff54b324b1"
+                }
+            ]
+        },
+        {
+            "name": "antimicro",
+            "buildsystem": "cmake",
```

```suggestion
            "buildsystem": "cmake-ninja",
```

This may or may not work but is a tad faster to build.